platformio.ini

```diff
@@ -15,6 +15,15 @@
 ; development use VSCode to change the target to a non-default
 ; by clicking on the target name in the bottom status bar.
 [platformio]
+; include custom environments to be build
+extra_configs =
+	./CustomDevices/_template/MyCustomDevice_platformio.ini
+	./CustomDevices/KAV_Simulation/EFIS_FCU/EFIS_FCU_platformio.ini
+	./CustomDevices/Mobiflight/GNC255/GNC255_platformio.ini
+	./CustomDevices/Mobiflight/TM1637/TM1637_platformio.ini
+	./CustomDevices/Mobiflight/GenericI2C/GenericI2C_platformio.ini
+	./CustomDevices/_all_CustomDevices/all_devices_platformio.ini
+	
 
```

Comment:
Include builts from external files which are part of the custom device repo and cloned.
For new custom devices onyl one entry has to be added. No more changes on this repo are necessary.

Reply:
Are any of these actually going to ship with the main mobiflight firmware? If so, where is the build step that clones them?